### ISSUE TITLE
Description overwrites other card's description on selection

### ISSUE THREAD

**Issue by r1ckdu** [nextcloud/deck#2579]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Create two Cards
2. Edit description of first card
3. While autosaving click on second card
4. Both cards have the same description now.
(This is also the case when the second card already has a description)

### Expected behaviour
Saving should be done before changing views.

### Actual behaviour
The description of your current selected card is copied to a new card. There is no way back.

## Server configuration detail

**Operating system:** Linux 4.4.213-rockchip64 #7 SMP Mon Feb 17 09:37:05 CET 2020 aarch64

**Webserver:** Apache (fpm-fcgi)

**Database:** mysql 10.3.25

**PHP version:** 

7.3.19-1~deb10u1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, sodium, standard, cgi-fcgi, mysqlnd, PDO, xml, bcmath, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, intl, json, ldap, exif, mysqli, pdo_mysql, Phar, posix, readline, redis, shmop, SimpleXML, smbclient, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, libsmbclient, Zend OPcache

**Nextcloud version:** 19.0.4 - 19.0.4.2

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** nextcloudpi

<details><summary>Signing status</summary>

Array()

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.5.0
 - activity: 2.12.1
 - calendar: 2.0.4
 - cloud_federation_api: 1.2.0
 - comments: 1.9.0
 - contacts: 3.4.1
 - contactsinteraction: 1.0.0
 - cookbook: 0.7.6
 - dav: 1.15.0
 - deck: 1.2.0
 - federatedfilesharing: 1.9.0
 - federation: 1.9.0
 - files: 1.14.0
 - files_pdfviewer: 1.8.0
 - files_rightclick: 0.16.0
 - files_sharing: 1.11.0
 - files_trackdownloads: 1.8.0
 - files_trashbin: 1.9.0
 - files_versions: 1.12.0
 - files_videoplayer: 1.8.0
 - firstrunwizard: 2.8.0
 - issuetemplate: 0.7.0
 - logreader: 2.4.0
 - lookup_server_connector: 1.7.0
 - news: 14.2.2
 - nextcloud_announcements: 1.8.0
 - notes: 3.6.4
 - notifications: 2.7.0
 - oauth2: 1.7.0
 - password_policy: 1.9.1
 - photos: 1.1.0
 - previewgenerator: 2.3.0
 - privacy: 1.3.0
 - provisioning_api: 1.9.0
 - recommendations: 0.7.0
 - serverinfo: 1.9.0
 - settings: 1.1.0
 - sharebymail: 1.9.0
 - support: 1.2.1
 - systemtags: 1.9.0
 - tasks: 0.13.6
 - text: 3.0.1
 - theming: 1.10.0
 - twofactor_backupcodes: 1.8.0
 - updatenotification: 1.9.0
 - viewer: 1.3.0
 - workflowengine: 2.1.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - files_texteditor
 - gallery
 - maps
 - nextcloudpi
 - previewgenerator.orig
 - survey_client
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": {
        "0": "localhost",
        "11": "***REMOVED SENSITIVE VALUE***",
        "1": "***REMOVED SENSITIVE VALUE***",
        "5": "nextcloudpi.local",
        "7": "nextcloudpi",
        "8": "nextcloudpi.lan",
        "12": "loemkers.com",
        "2": "loemkers.com"
    },
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "19.0.4.2",
    "overwrite.cli.url": "https:\/\/loemkers.com\/",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0,
        "timeout": 0,
        "password": "***REMOVED SENSITIVE VALUE***"
    },
    "tempdirectory": "\/media\/1c2e55ef-caf9-4713-a9f0-9082082ed80c\/ncdata\/tmp",
    "mail_smtpmode": "sendmail",
    "mail_smtpauthtype": "LOGIN",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "preview_max_x": "2048",
    "preview_max_y": "2048",
    "jpeg_quality": "60",
    "overwriteprotocol": "https",
    "maintenance": false,
    "data-fingerprint": "ddadc32c5e34b32794e470cc689d24f7",
    "theme": "",
    "logfile": "\/media\/1c2e55ef-caf9-4713-a9f0-9082082ed80c\/ncdata\/nextcloud.log",
    "loglevel": "2",
    "log_type": "file",
    "app_install_overwrite": [
        "gallery"
    ],
    "mail_sendmailmode": "smtp"
}
```
</details>

**Are you using external storage, if yes which one:** 

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** 

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:82.0) Gecko/20100101 Firefox/82.0

**Operating system:** 

## Logs
No Logs

**Comment by stefan-niedermann** [comment]
This has been fixed in Deck 1.2.1 which is already available, please update to the latest app version 🙂

**Comment by stefan-niedermann** [comment]
Duplicate of https://github.com/nextcloud/deck/issues/2568 (fixed by https://github.com/nextcloud/deck/issues/2568)

@r1ckdu do you always like your own issues? :smile:

**Comment by r1ckdu** [comment]
@stefan-niedermann 
Don't you?

Sorry about this misposting. I did not check the closed issues. Happy to hear there is a fix 👍

**Comment by stefan-niedermann** [comment]
> Don't you?

Only if i *really* desire a feature, but in this case i usually also put a :rocket:, :tada: and :heart: next to it :wink: 

> Sorry about this misposting.

No worries, we are happy about each bug report and rely on feedback from the users :)
